### ISSUE TITLE
Site 'Help' system 

### ISSUE THREAD

**Issue by tim-deighton** [kendraio/kendraio-app#64]
The site needs to provide inline/in-page help to the user.

**Comment by darrenmothersele** [comment]
Yes, it would be interesting to see how additional help could be added to adapters/configs and how this would be used within the app.
